src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage{
+    By emailFieldLocator = By.cssSelector("input[type='email']");
+    By passwordFieldLocator = By.cssSelector("[type='password']");
+    By submitButtonLocator = By.cssSelector("button[type='submit']");
+    By registerButtonLocator = By.cssSelector("input[value='Register']");
+    By registrationFieldLocator = By.cssSelector("#hel");
+    String email = "vlina7078@yahoo.com";
+    String password = "te$t$tudent";
```

Comment:
You should use encapsulation of non-static and non-final fields.

I recommend:  make all the **locators private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");